invio/umbrel-app.yml

```
@@ -0,0 +1,28 @@
+manifestVersion: 1
+id: invio
+category: finance
+name: Invio
+version: "1.7.0"
+tagline: Self-hosted invoicing without the bloat.
```

remove `.` to fix the error of tagline
```suggestion
tagline: Self-hosted invoicing without the bloat
```